server/protocols/jmap/src/main/java/org/apache/james/jmap/model/mailbox/Rights.java

```diff
@@ -190,6 +211,28 @@ public Rights(Multimap<Username, Right> rights) {
         return rights.asMap();
     }
 
+    public MailboxACL toMailboxAcl() {
+        BinaryOperator<MailboxACL> union = Throwing.binaryOperator(MailboxACL::union);
+
+        return rights.asMap()
+            .entrySet()
+            .stream()
+            .map(entry -> new MailboxACL(
```

Comment:
we lack a more fluent constructor for MailboxACL that would not require building a map

Reply:
We have...